test/unit/payments/onchain-payment-flow-builder.spec.ts

```diff
@@ -287,21 +304,24 @@ describe("OnChainPaymentFlowBuilder", () => {
 
                 const paymentLowest = await withBtcWalletBuilder
                   .withoutRecipientWallet()
-                  .withAmount(51) // Close to 1 cent
+                  .withAmount({ amount: BigInt(51), currency: amountCurrency }) // Close to 1 cent
```

Comment:
I think you can just use `51n` or some other postfix like that to signify `BigInt`